src/unistd.rs

```diff
@@ -735,6 +735,27 @@ pub fn gethostname<'a>(buffer: &'a mut [u8]) -> Result<&'a CStr> {
     })
 }
 
+/// Get a CStr containing the name of the user logged in on the
+/// controlling terminal of the process
+/// ([getlogin(3)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/getlogin.html)).
+///
+/// The posix specification says that "the string is statically
+/// allocated and might be invalidated or overwritten on subsequent
+/// calls to this function", so you may want to copy the value before
+/// using it.
+///
+/// ```
```

Comment:
This should go in an Examples section. See the [Rust docs about this](https://doc.rust-lang.org/book/first-edition/documentation.html#special-sections).

Reply:
Added it to "Example" because that was more common in the rest of the file, and removed the `no_run` from `gethostname` because we might as well be consistent.